### ISSUE TITLE
Pods URL Field not working

### ISSUE THREAD

**Issue by simonclay** [elementor/elementor#7564]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Install Pods and set up a new Pod with URL a field. Then on an Elementor widget (eg. Image) ...

Link > Custom URL > Dynamic > Pods URL Field > (select the field you made). Then click update. There is no link applied to the image.

**Workaround:** Link > Custom URL > Dynamic > Shortcode > [pods field="my_custom_link"]

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v5.5.59-MariaDB-38.11
	PHP Version: 7.2.16
	PHP Max Input Vars: 1000
	PHP Max Post Size: 16M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.1.1
	Site URL: https://universal-care.developmentandtesting.co.uk
	Home URL: https://universal-care.developmentandtesting.co.uk
	WP Multisite: No
	Max Upload Size: 16 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: Europe/London
	Debug Mode: Inactive

== Theme ==
	Name: Astra Child
	Version: 1.0.0
	Author: Brainstorm Force
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 1.7.2
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0.3 Safari/605.1.15

== Active Plugins ==
	All-in-One WP Migration
		Version: 6.87
		Author: ServMask

	All-in-One WP Migration Unlimited Extension
		Version: 2.26
		Author: ServMask

	Astra Pro
		Version: 1.7.1
		Author: Brainstorm Force

	Classic Editor
		Version: 1.4
		Author: WordPress Contributors

	Ele Custom Skin
		Version: 1.0.9
		Author: Liviu Duda

	Elementor
		Version: 2.5.9
		Author: Elementor.com

	Elementor Pro
		Version: 2.4.8
		Author: Elementor.com

	iThemes Security
		Version: 7.3.2
		Author: iThemes

	Jet Elements For Elementor
		Version: 1.15.5
		Author: Zemez

	Lenix Elementor Leads addon
		Version: 1.5
		Author: Lenix

	Pods - Custom Content Types and Fields
		Version: 2.7.12
		Author: Pods Framework Team

	Ultimate Addons for Elementor
		Version: 1.10.0
		Author: Brainstorm Force

	WordPress Importer
		Version: 0.6.4
		Author: wordpressdotorg

	Yoast SEO
		Version: 10.0.1
		Author: Team Yoast


== Log ==
	: 
Log: showing 13 of 132019-01-27 20:00:07 [info] elementor::elementor_updater Started 
2019-01-27 20:00:07 [info] Elementor/Upgrades - _v_2_3_0_widget_image Start  
2019-01-27 20:00:07 [info] Elementor/Upgrades - _v_2_3_0_widget_image Finished 
2019-01-27 20:00:07 [info] Elementor/Upgrades - _v_2_3_0_template_type Start  
2019-01-27 20:00:07 [info] Elementor/Upgrades - _v_2_3_0_template_type Finished 
2019-01-27 20:00:07 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.2.7',
  'to' => '2.4.4',
)]
2019-01-27 20:00:07 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '2.2.7',
  'to' => '2.4.4',
)]
2019-02-16 15:29:14 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.4',
  'to' => '2.4.6',
)]
2019-02-16 15:29:47 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.1.11',
  'to' => '2.4.4',
)]
2019-03-12 11:17:38 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.6',
  'to' => '2.5.5',
)]
2019-03-12 11:20:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.4',
  'to' => '2.4.8',
)]
2019-03-15 12:35:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.5',
  'to' => '2.5.8',
)]
2019-03-19 16:10:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.8',
  'to' => '2.5.9',
)]

JS: showing 1 of 1JS: 2019-03-14 12:03:31 [error X 15][https://universal-care.developmentandtesting.co.uk/wp-admin/load-scripts.php?c=0&load%5B%5D=underscore,jquery-core,jquery-migrate,backbone,shortcode,utils,wp-util,wp-backbone,media-models,moxiejs,plupload,wp-plupload,jqu&load%5B%5D=ery-ui-core,jquery-ui-widget,jquery-ui-mouse,jquery-ui-sortable&ver=5.1.1:5:30810] undefined is not an object (evaluating \'c.pageYOffset\') 

PHP: showing 1 of 1PHP: 2019-03-19 16:11:35 [notice X 78][/var/www/vhosts/developmentandtesting.co.uk/universal-care.developmentandtesting.co.uk/wp-content/plugins/elementor/core/files/css/base.php::703] Undefined index: _id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


				
```
</details>

**Comment by simonclay** [comment]
It seems that the issue is when there is just one 'Key' to select. If there is only 1 Key to select from, it will not register, but if there are 2 Keys to chose from, it can register the Key.

**Comment by simonclay** [comment]
It is not limited to the Pods URL fieldHere is a visual 

Not able to select...
![elementor_pods_date_not_working](https://user-images.githubusercontent.com/2143351/54775142-39378c80-4c05-11e9-8889-d4c44f07c565.gif)

Able to select when another item is in the Key list...
![elementor_pods_date_working](https://user-images.githubusercontent.com/2143351/54775156-3dfc4080-4c05-11e9-8840-56511bc0716a.gif)

**Comment by bainternet** [comment]
@simonclay 

Which browser are you using? does it happen in all browsers?

**Comment by simonclay** [comment]
@bainternet Thanks for raising that question. It appears that is is an issue in Safari. Chrome and Firefox work as expected. (Mac)

Unlike Chrome and Firefox, Safari pre-selects the only key available, which seems to prevent the key from being activated. Chrome and Firefox wait for you to select the Key from the dropdown, which then works fine.

**Comment by KingGrizzly** [comment]
+1 on this issue. Even with multiple keys to choose from, no link is applied to item. I've tried this as a dynamic link for images and text. Both fail.

<img width="340" alt="Screen Shot 2019-04-09 at 10 50 17 PM" src="https://user-images.githubusercontent.com/48737505/55852362-e1ac8280-5b19-11e9-87f0-ad1405e2e3fd.png">

**Comment by bainternet** [comment]
Closing as fixed in Elementor Pro 2.5.10 - 2019-05-28


Thanks.

**Comment by srack835** [comment]
I am experiencing this same issue, a year later. This affects URL and Phone fields.
